Automattic/wp-calypso
# A/B Test: Wrapping up Concierge Quickstart test and implementing variant

#### Changes proposed in this Pull Request

* Removed `ConciergeSessionNudge` component and leave `ConciergeQuickstartSession` as new control.

#### Testing instructions

Start Calypso locally with this branch and visit http://calypso.localhost:3000/.

1. Create a new site (to make sure the last redirect is consistent, **don't** choose "_Sell products or collect payments_" for the "_What’s the primary goal..._" question) or use an existing test site.

2. Purchase or upgrade to a plan (no domain).

3. After purchase, confirm you're shown the Concierge upsell page with the `Quick Start` copy.

![image](https://user-images.githubusercontent.com/538790/56048046-f3bf3880-5d46-11e9-92ad-cf4983401b4d.png)

4. Click the button: "Yes, I want a WordPress Expert by my side". You should be taken to the checkout with the session added to the cart.

<img width="292" alt="Screen Shot 2019-04-10 at 13 03 25" src="https://user-images.githubusercontent.com/538790/55875499-861fda80-5b95-11e9-9ed3-39d0b8e1ebf7.png">

7. Purchase the session.

8. Confirm you're either taken to the checklist or to the Thank you page.

<img width="851" alt="screen shot 2019-01-02 at 11 21 46 am" src="https://user-images.githubusercontent.com/690843/50608428-81a2a200-0e81-11e9-9234-07da29eb5c98.png">

<img width="1009" alt="Screen Shot 2019-04-10 at 13 38 59" src="https://user-images.githubusercontent.com/538790/55875726-15c58900-5b96-11e9-8004-8fe0bbe9fe80.png">
